packages/api-client-core/package.json

```diff
@@ -41,7 +41,7 @@
     "@types/node": "^14.18.26",
     "conditional-type-checks": "^1.0.6",
     "gql-tag": "^1.0.1",
-    "jest": "^27.5.1",
+    "jest": "^28.0.8",
```

Comment:
When developing locally this was complaining of mismatched peer dependencies with ts-jest